src/main/java/com/programmers/customer/domain/Customer.java

```diff
@@ -0,0 +1,50 @@
+package com.programmers.customer.domain;
+
+import java.time.LocalDateTime;
+import java.util.UUID;
+
+public class Customer {
+
+    private static final String CUSTOMER_NULL_MESSAGE = "[ERROR] 유효하지 않은 Customer Id 입니다. (null)";
+
+    private final UUID customerId;
+    private final String name;
+    private final LocalDateTime createdAt;
+    private final LocalDateTime modifiedAt;
+
+    public Customer(UUID customerId, String name, LocalDateTime createdAt) {
+        validateCustomerId(customerId);
+        this.customerId = customerId;
+        this.name = name;
+        this.createdAt = createdAt;
+        this.modifiedAt = null;
```

Comment:
```suggestion
        this(customerId, name, createdAt, null);
```
- 이렇게 하면 코드 중복이 줄어들겠네요!

- `modifiedAt`이 있는 생성자와 없는 생성자는 어떤 차이가 있는걸까요?